tests/Doctrine/Tests/ORM/Functional/SchemaTool/SqliteSchemaToolTest.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\Tests\ORM\Functional\SchemaTool;
+
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\ORM\Annotation as ORM;
+use Doctrine\ORM\EntityManagerInterface;
+use Doctrine\ORM\Tools\SchemaTool;
+use Doctrine\Tests\OrmTestCase;
+use Doctrine\Tests\TestUtil;
+
+class SqliteSchemaToolTest extends OrmTestCase
```

Comment:
Please name this test class, and all related classes, after the issue it fixes (`GH7841Test`, `GH7841Parent`, `GH7841Child`) and add a `@group GH7841` annotation to the test class.